camera_calibration/src/camera_calibration/calibrator.py

```diff
@@ -938,6 +939,22 @@ def pt2line(x0, y0, x1, y1, x2, y2):
         else:
             return None
 
+    def update_mask(self, img, screen):
+        """
+        Update or if None create a red mask with the same size of screen
+        and add on top of it detection areas displayed through green circles
```

Comment:
Add a comment what is this mask used for -> visualization of covered image area with good corner detection
red-> not covered, green->covered
Also you are missing docstring of arguments, please add